### DISCUSSION TITLE
An Astro plugin to open external links in a new tab

### DISCUSSION THREAD

**Discussion by daniele-salvagni** [daniele-salvagni/dan.salvagni.io#12] [Comments]
**Comments for the blog post:** https://dsalvagni.com/b/astro-plugin-open-external-links-in-new-tab/

**Comment by AdnanRahmanpoor** [comment]
Just what I needed for my blog site. Worked Flawlessly.

Thanks a lot.

**Comment by razvantimis** [comment]
Thanks!

**Comment by tjex** [comment]
perfect 🙌 thanks for this.

**Comment by PermeationLoop** [comment]
Copied, Thank you!
